net/src/ipc_net_worker.rs

```diff
@@ -103,14 +108,42 @@ impl IpcNetWorker {
         if config["socketType"] != "zmq" {
             bail!("unexpected socketType: {}", config["socketType"]);
         }
+        let mut block_connect = true;
+        if let Some(b) = config["blockConnect"].as_bool() {
+            block_connect = b;
+        }
```

Comment:
Maybe do:
```rust
let block_connect = config["blockConnect"].as_bool().unwrap_or(true);
```

Reply:
Updated. I totally appreciate your suggestion, and the terseness of the change, and at the same time... if I'm not allowed to use if let or match statements anymore, I wish someone would add a cli flag to the compiler or something to disallow them in the first place to prevent the back-n-forth...